packages/kbn-test/src/jest/utils/testbed/testbed.ts

```diff
@@ -251,6 +251,14 @@ export const registerTestBed = <T extends string = string>(
         component.update();
       };
 
+      const setSearchBarValue: TestBed<T>['form']['setSearchBarValue'] = (testSubject, value) => {
+        const searchBar = find(testSubject);
+        if (!searchBar.length) {
+          throw new Error(`"${testSubject}" was not found.`);
+        }
+        searchBar.simulate('keyup', { key: 'Enter', target: { value } });
```

Comment:
From reading this function, there's nothing that leads me to deduce that it's specifically designed to interact with EuiSearchBar's input. Can we add a comment to explain this?

Reply:
Thanks for this suggestion, @cjcenizal! I added a doc comment to the method.